user_docs/en/userGuide.t2t

```diff
@@ -2001,7 +2001,8 @@ Portable versions of NVDA store all settings and add-ons in a directory called u
 
 Installed versions of NVDA store all settings and add-ons in a special NVDA directory located in your Windows user profile.
 This means that each user on the system can have their own NVDA settings.
-To get to your settings directory for an installed version of NVDA, on the start menu you can go to programs -> NVDA -> explore user configuration directory.
+To open your settings directory  from anywhere you can use [Input Gestures dialog #InputGestures] to add a custom gesture.

```

Comment:
There is a double space here (after 'directory'):
```suggestion
To open your settings directory from anywhere you can use [Input Gestures dialog #InputGestures] to add a custom gesture.
```

Have you built this to check that the link works?